### ISSUE TITLE
Can't load Soundcloud music "This url does not appear to be a playable track"

### ISSUE THREAD

**Issue by MetalMallard** [sedmelluq/lavaplayer#573]
Whenever i try to play a soundcloud music i'm getting the "This url does not appear to be a playable track" error

error =>
```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: This url does not appear to be a playable track.
	at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudHtmlDataLoader.load(DefaultSoundCloudHtmlDataLoader.java:31) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.loadFromSet(DefaultSoundCloudPlaylistLoader.java:73) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.load(DefaultSoundCloudPlaylistLoader.java:61) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioSourceManager.loadItem(SoundCloudAudioSourceManager.java:119) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419) ~[lavaplayer-1.3.62.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218) ~[lavaplayer-1.3.62.jar!/:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
```
using last version of Lavalink

**Comment by MCOfficer** [comment]
Duplicate of #569. Lavalink still uses 1.6.61, the fix is in 1.3.64 or newer.

**Comment by Frederikam** [comment]
> using last version of Lavalink

Use one of the dev builds

**Comment by MetalMallard** [comment]
I used the last one from the CI server! isn't it a dev build ?

**Comment by MetalMallard** [comment]
> Duplicate of #569. Lavalink still uses 1.3.61, the fix is in 1.3.64 or newer.

Thanks didn't see ! 

> > using last version of Lavalink
> 
> Use one of the dev builds
> 
> Edit: Actually, I've just tagged a new release.
> https://github.com/Frederikam/Lavalink/releases/tag/3.3.2.3

will do thanks !
